saleor/graphql/shipping/mutations/shippings.py

```diff
@@ -372,6 +371,14 @@ def delete_invalid_shipping_zone_to_warehouse_relation(cls, shipping_zone):
             id__in=shipping_zone_warehouses_to_delete
         ).delete()
 
+    @classmethod
+    def _extend_shipping_zone_countries(cls, data):
+        countries = get_countries_without_shipping_zone()
+        try:
+            data["countries"].extend([country for country in countries])
+        except (KeyError, AttributeError):
```

Comment:
I think this is OK but wasn't the previous solution with `setdefault` better? It looked cleaner like we are able to determine what data type we're dealing with and we don't have to guess and handle it by trial and error. Anyway, if others find it OK, it's not a big deal for me 😄

Reply:
I also think that previous solution with `setdefault` was much cleaner. However I changed it to this one as previous solution wasn't working in case where `countries` was already in data and where not list type e.g. sending null.